rcsb/db/mongo/PdbxLoader.py

```diff
@@ -952,7 +953,7 @@ def loadWorker(self, dataList, procName, optionsD, workingDir):
             #
             ok = len(failContainerIdS) == 0
             self.__end(startTime, procName + " with status " + str(ok))
-            return retList, [], []
+            return retList, list(cardinalIdFailS), []
```

Comment:
This gets returned as `failListT` on line 724: https://github.com/rcsb/py-rcsb_db/blob/cda61788a26644f09eaf9355d7e050b4253c8301/rcsb/db/mongo/PdbxLoader.py#LL724C8-L724C8, which is then used to assign the value of `ok`.

Reply:
I see. The issue I'm having is following these return values thought the MultiProcUtil.runMulti() call because the order is swapped with `retList`. I think some comments explaining what's going on would be really helpful in that method (doesn't have to be done now) because the variable names are not very descriptive.

Reply:
Thank you for questioning that! That ended up leading me to realize I was looking at it wrong. The `MultiProcUtil` methods are very confusing. I've changed the code to correct this and updated the DocStrings and comments to try to explain how it works...hope that makes it clearer to you as well.